CHANGES/README.rst

```diff
@@ -1,15 +1,12 @@
 .. _Adding change notes with your PRs:
 
-Adding change notes with your PRs
```

Comment:
Keep this title and drop the one in `contributing.rst` instead. Follow what I did in ansible/pylibssh.

Reply:
This looks more like an intro paragraph, and then everything else are subheadings. It looks a bit weird to me that these headings are on the same level (including when I look at https://ansible-pylibssh.readthedocs.io/en/latest/contributing/guidelines/)

Reply:
Second level headings on that page are:

- Contributing docs
- Adding change notes with your PRs
- Alright! So how do I add a news fragment?

Reply:
I've readded the header, but at a different level. Let me know if that's OK.

Reply:
Couldn't get that to build correctly, so have tweaked it again. I suggest we use the standard described at https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#sections
With the first heading being a section (=) and the others being subsections (-). Any pages which want to include the file will need to adopt that standard.

Reply:
Dunno. From the RST standpoint, it shouldn't matter. But it's nice to have some consistency. OTOH, I'd rather implement the header level changes in a separate PR, then here. I'd be fine with having the same-level headers for a while. Though, if we go for the Sphinx-suggested semantics, I'd extend that with documenting such a decision explicitly and maybe tried adding some linting for consistency. But then, it becomes a bigger standalone effort.

IIRC I picked up parts of that structure from https://setuptools.pypa.io/en/latest/development/developer-guide.html#making-a-pull-request (but I also contributed back so I don't remember exactly where each specific sub-idea was coming from). It's one of the projects I wanted to keep in sync with the doc structure. There's also frozenlist/yarl/multidict, of course. And some others, in other orgs.

Currently, I'd like the document to diverge minimally, since it allows to copy parts over with less merge conflicts and cleaner diffs.

Reply:
> I picked up parts of that structure from https://setuptools.pypa.io/en/latest/development/developer-guide.html#making-a-pull-request

Right, but that one has a parent header, which has been removed before inserting in another page. We could readd that header with the 2 sentence intro (although seems a little redundant to me).

Reply:
I kinda wanted the header in the docs folder. But maybe that's fine too.

Reply:
In the interests of time, I'm going to merge this, but feel free to follow up with any other changes. It shouldn't be important if they miss this release.

Reply:
Fair